stuff/pipelines/rabbitmq_connection.py

```diff
@@ -19,8 +19,12 @@ def connect(
     username = args.rabbitmq_username
     password = args.rabbitmq_password
     credentials = pika.PlainCredentials(username, password)
-    parameters = pika.ConnectionParameters('rabbitmq', 5672, '/',
-                                           credentials, heartbeat=600)
+    parameters = pika.ConnectionParameters('rabbitmq',
+                                           5672,
+                                           '/',
+                                           credentials,
+                                           heartbeat=600,
+                                           blocked_connection_timeout=300.0)
```

Comment:
```suggestion
                                           # mypy does not support structural typing yet
                                           # https://github.com/python/mypy/issues/3186
                                           blocked_connection_timeout=300.0, # type: ignore:
    )
```
pues resulta que es algo que mypy no soporta :'( por eso no podemos tener cosas bonitas.

Reply:
Listo